specification/metrics/semantic_conventions/system-metrics.md

```diff
@@ -39,6 +39,7 @@ instruments not explicitly defined in the specification.
 |                        |                                                                                                          |       |                                                   |            | cpu              | CPU number [0..n-1]                 |
 | system.cpu.utilization | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of CPUs | 1     | Gauge                                             | Double     | state            | idle, user, system, interrupt, etc. |
 |                        |                                                                                                          |       |                                                   |            | cpu              | CPU number (0..n)                   |
+| system.cpu.count       | Reports the and number of available CPUs - may changes over time                                         | 1     | UpDownCounters                                    | Int64      | device           | (identifier e.g. `machine-id`)      |
```

Comment:
I think we should clarify if this is the number of physical or logical processors

Reply:
I guess we probably want to report both, or at least account for the possibility of wanting to do so in the future, so we may want to make the name reflective of which one of the two we are reporting (so maybe having `physical` on the name somewhere makes sense, unless it's clear that when people talk about CPUs they are talking about physical ones?))

Reply:
hm.. what would you recommend? Something like `system.cpu.vcount` and `system.cpu.ncount`?

cc @chambridge

Reply:
> I think we should clarify if this is the number of physical or logical processors

@mx-psi can you clarify what the use case of this is? As a user of the `hostmetricsreceiver` anecdotally wouldn't need this to be split up. From an instrumentation standpoint, is it possible to detect if CPUs are virtual or physical?